### PR TITLE
@jakmro/classification ios

### PR DESCRIPTION
**Pull request by jakmro**
## Description
Image classification for iOS 

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (improves or adds clarity to existing documentation)

### Tested on
- [x] iOS
- [ ] Android

### Checklist
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly
- [x] My changes generate no new warnings

### CODE REVIEW

File: examples/computer-vision/assets/classification/android/efficientnet_v2_s_xnnpack.pte
Comment (jakmro) [inline examples/computer-vision/assets/classification/android/efficientnet_v2_s_xnnpack.pte:1]: This model will be hosted on Hugging Face

File: examples/computer-vision/assets/classification/ios/efficientnet_v2_s_coreml_all.pte
Comment (jakmro) [inline examples/computer-vision/assets/classification/ios/efficientnet_v2_s_coreml_all.pte:1]: This model will also be hosted on Hugging Face

File: examples/computer-vision/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "metro-config": "^0.81.0",
```
Comment (mkopcins) [inline examples/computer-vision/package.json]: either revert or write some instructions in readme how to build react-native-executorch-0.1.100.tgz

File: src/models/Classification.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  isModelReady: boolean;
+  isModelGenerating: boolean;
+  forward: (input: string) => Promise<{ [category: string]: number }>;
+}
+
+export const useClassification = ({
+  modulePath,
+}: Props): ClassificationModule => {
+  const [error, setError] = useState<null | string>(null);
+  const [isModelReady, setIsModelReady] = useState(true);
```
Comment (mkopcins) [inline src/models/Classification.ts]:
```suggestion
  const [isModelReady, setIsModelReady] = useState(false);
```

File: src/models/Classification.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      }
+    };
+
+    loadModel();
+  }, [modulePath]);
+
+  const forward = async (input: string) => {
+    if (!isModelReady) {
+      throw new Error(getError(ETError.ModuleNotLoaded));
+    }
```
Comment (mkopcins) [inline src/models/Classification.ts:48]: add check for `isModelGenerating`

File: ios/RnExecutorch/models/classification/Constants.h [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#import <string>
+
+extern const std::string imagenet1k_v1_labels[];
```
Comment (mkopcins) [inline ios/RnExecutorch/models/classification/Constants.h:3]: Imo better do constexpr and define it here instead of mm file

**Review by mkopcins — CHANGES_REQUESTED** [review]

**Review by mkopcins — APPROVED** [review]

**Pull request merged**